actor-tests/src/test/java/org/apache/pekko/util/JavaConverterScala3InlineTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
```

Comment:
this  is a new File,can we remove the description about akka

Reply:
@pjfanning Whats your views on this, original Apache header also makes sense but this header is the one copied from https://github.com/apache/incubator-pekko/blob/b0fdac259bd57fdd481483f3fe9a7aec6e1ff38a/actor-tests/src/test/scala/org/apache/pekko/util/Scala212CompatTest.scala#L1-L8 which is a comparable new test that was written

Reply:
A copy should keep the lightbend header

Reply:
This isn't a copy, it's a new source file

Reply:
If the content is new ,then only asf header

Reply:
brand new files should use the standard ASF header - unless it has code copied in from an existing file in which case it should use the headers from that file

Reply:
If anything Scala212CompatTest seems to have the wrong header - that file looks like it was written as a fresh set of tests by the Pekko team.

Reply:
> If anything Scala212CompatTest seems to have the wrong header - that file looks like it was written as a fresh set of tests by the Pekko team.

Thanks, ill fix the header for that file in another PR as well as fixing the headers in this PR